lms/static/styles/components/_Table.scss

```diff
@@ -21,6 +21,15 @@ $table-item-height: 40px;
   }
 }
 
+.Table__spinner {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
```

Comment:
I'm thinking that standardizing the Spinner component/pattern might be useful in the near term in the shared package?

Reply:
Yes, I think so.